search-service/src/main/kotlin/com/egm/stellio/search/service/TemporalEntityAttributeService.kt

```diff
@@ -461,7 +461,7 @@ class TemporalEntityAttributeService(
             queryTerm.contains("<") ->
                 Triple(queryTerm.split("<")[0], "<", queryTerm.split("<")[1])
             queryTerm.contains("=~") ->
-                Triple(queryTerm.split("=~")[0], "=~", queryTerm.split("=~")[1])
+                Triple(queryTerm.split("=~")[0], "like_regex", queryTerm.split("=~")[1])
```

Comment:
did you check the compatibility of the `like_regex` operator with the regex format that should be supported by NGSI-LD implementations?